src/service/poc_mobile.proto

```diff
@@ -56,6 +56,28 @@ message cell_heartbeat_ingest_report_v1 {
   cell_heartbeat_req_v1 report = 2;
 }
 
+message wifi_heartbeat_req_v1 {
+  // Public key of the hotspot
+  bytes pub_key = 1;
+  // Timestamp of heartbeat in seconds from unix epoch
+  uint64 timestamp = 2;
+  double lat = 3;
+  double lon = 4;
+  // Timestamp of location validation in seconds from unix epoch
+  uint64 location_validation_timestamp = 5;
+  bool operation_mode = 6;
+  bytes coverage_object = 7;
```

Comment:
why is this bytes  and not a coverage_object type ?

Reply:
this is the uuid of the coverage_object in bytes just like in the cell_heartbeat https://github.com/helium/proto/blob/302880a665f92db5ac865a1878444ec87d1a80e6/src/service/poc_mobile.proto#L47

Reply:
k that makes sense then